### PR TITLE
Features/test in tox

### PR DESCRIPTION
**Pull request by Mikanebu**
* Added `setup.py`
* Added `tox.ini`
* Added `pylama.ini`
* Added `MANIFEST.in`
* Added `makefile`
* Configured `travis.yml` for make
* Added `bitstore/VERSION` 
* Updated README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,17 @@ An API server for managing a Source Spec Registry
 
 ## Quick start
 
-- Clone the repo, 
-- install dependencies from `requirements.txt`, 
-- and run the server (`server.py`)
+# Clone the repo and install
```
Comment (zelima) [inline README.md]: Think this will look better with h3 headings

Comment (Mikanebu) [inline README.md]: ok

**Comment by zelima** [comment]
Looks fine

File: README.md (hunk 1 above)
Comment (zelima) [inline README.md]: Do thisdirectly to the master As I merged

Comment (Mikanebu) [inline README.md]: ok

**Pull request merged**